src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +22,37 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val listOfGreetings: MutableList<String> = mutableListOf()
+
+        val englishImplementation = object : HelloWorldGreeting {
+            override fun greet(): String = ENGLISH_GREETING
+
+            override fun greetSomeone(someone: String): String = String.format(Locale.ENGLISH,
```

Comment:
String.format is OK, however in kotlin you may use this syntax "$ENGLISH_HI  $someone"